docs/docs/ias.md

```diff
@@ -44,7 +52,7 @@ enclave.start(
 });
 ```
 
-!!! important
-    Ensure you have
-    [installed the kernel driver and system software](machine-setup.md#install-the-kernel-driver-and-system-software)
+!!!Important
+    
+    [Install the kernel driver and system software](non-cloud-deployment.md#install-the-kernel-driver-and-system-software)
     on your machine before trying to use EPID attestation in debug or release mode.
```

Comment:
I would simplify here: `before using` instead of `before trying to use`

Reply:
Done.